### PR TITLE
Test/patrol setup

### PR DESCRIPTION
**Pull request by KmlBaranowski**
Setting up Patrol tool for future tests

### CODE REVIEW

File: integration_test/src/config.dart
Comment (jBorkowska) [inline integration_test/src/config.dart:1]: I don't think you need this file, these configs are created by default when you call `patrolTest`

File: integration_test/startapp_test.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'src/common_methods.dart';
+import 'package:patrol/patrol.dart';
+
+void main() {
+  patrolTest(nativeAutomation: true, 'Starting application', ($) async {
```
Comment (jBorkowska) [inline integration_test/startapp_test.dart]: We have a convention that test description is always first argument, and test callback is last, so put `nativeAutomation` in the middle

**Review by jBorkowska — CHANGES_REQUESTED** [review]

File: integration_test/src/config.dart
Comment (KmlBaranowski) [reply integration_test/src/config.dart:1]: Indeed successful test execution and application building do not require this file, i've deleted this

**Review by jBorkowska — APPROVED** [review]

File: integration_test/startapp_test.dart (hunk 1 above)
Comment (KmlBaranowski) [reply integration_test/startapp_test.dart]: I've corrected the arguments order so that test description would be first

**Pull request merged**